File: Java/bee 1015.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double x1, y1, x2, y2;
        x1 = sc.nextDouble();
        y1 = sc.nextDouble();
        x2 = sc.nextDouble();
        y2 = sc.nextDouble();

        System.out.printf("%.4f\n", Math.sqrt(Math.pow(x1-x2,2) + Math.pow(y1-y2,2)));
    }
}

File: Kotlin/bee 1015.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import kotlin.math.pow
import kotlin.math.sqrt

fun main(args: Array<String>){
    val br = BufferedReader(InputStreamReader(System.`in`))
    val (x1, y1) = br.readLine().split(" ").map {it.toDouble()}
    val (x2, y2) = br.readLine().split(" ").map {it.toDouble()}

    println(String.format("%.4f", sqrt((x2-x1).pow(2)+(y2-y1).pow(2))))
}
